readthedocs/templates/builds/build_detail.html

```diff
@@ -56,7 +56,7 @@
 
       <div data-bind="visible: finished() && success()">
         <li>
-          <a class="" href="{{ build.version.get_absolute_url }}">
+          <a data-bind="attr: {href: docs_url}" href="{{ build.version.get_absolute_url }}">
```

Comment:
This data-bind was taken from http://knockoutjs.com/documentation/attr-binding.html.

Also I left the `href` in case someone has the js disable, but there is a condition first, and all the builds will be no show if someone doesn't have the js activated, so this can be deleted I guess.